src/main/java/roomescape/application/service/TossPaymentService.java

```diff
@@ -38,7 +39,8 @@ public void processPayment(TossPaymentRequestDto tossPaymentRequestDto,
         TossPaymentConfirmResponseDto tossPaymentConfirmResponseDto = tossPaymentWithHttpClient.requestConfirmation(
                 tossPaymentConfirmDto, requestKey);
 
-        if (!tossPaymentConfirmResponseDto.status().equals("DONE")) {
+        String status = tossPaymentConfirmResponseDto.status();
+        if (!PaymentStatus.isAcceptedStatus(status)) {
```

Comment:
변수 할당이 꼭 필요하지 않다면 피하는 습관을 만드는것도 좋습니다 :)
의미없는 리소스를 만들어서 좋을 것은 없으니까요 😃 
변수할당하면 가독성은 좋아지지만 
개인적으로 `가독성에 큰 차이가 없고 두 번이상 사용하지 않는다면` 변수할당을 지양하는 편이에요!

```suggestion
        if (!PaymentStatus.isAcceptedStatus(tossPaymentConfirmResponseDto.status())) {
```

Reply:
> 의미없는 리소스를 만들어서 좋을 것은 없으니까요 😃

저는 한번 더 뎁스가 타고 들어간다는 생각에 복잡해보일까봐 변수로 할당을 했었는데, 찰리 말씀에 동의해 수정해보았습니다! :)
앞으로 코드를 짤 때 한번씩 리소스 중점으로 고려해보려고 노력해보아야겠어요 ㅎㅎ

Reply:
[chore: LoginMember 가 파라미터에서 드러나지 않도록 수정](https://github.com/woowacourse/spring-roomescape-payment/pull/276/commits/c3aa377ad0bfbd0440a9a160290eaf0ae3b7b4c4)

Reply:
[del: 불필요한 변수 선언 삭제](https://github.com/woowacourse/spring-roomescape-payment/pull/276/commits/51d699efd6a2449783882b1aab31d71683df46df)